app/services/schools/advice/heating_control_service.rb

```diff
@@ -3,6 +3,18 @@ module Advice
     class HeatingControlService
       include AnalysableMixin
 
+      #number of days heating on in warm weather => rating
+      WARM_WEATHER_DAYS_RATING = {
+        0..6     => :excellent,
+        6..11    => :good,
```

Comment:
There looks like there is an overlapping range here. Should it be 0..5 => :excellent and 6..11 => :good (or similar?)